phylanx/execution_tree/primitives/base_primitive.hpp

```diff
@@ -283,6 +284,16 @@ namespace phylanx { namespace execution_tree { namespace primitives
             return 0;
         }
 
+        std::int64_t get_count()
```

Comment:
This function can be `const` (same is valid for `get_duration()` below).

Reply:
Done.